challenges/objects-arrays.js

```diff
@@ -91,6 +111,7 @@ The zoos want to display both the scientific name and the animal name in front o
 
 */
 const displayNames = [];
+zooAnimals.forEach((zooAnimal) => displayNames.push(zooAnimal.animal_name + '' +zooAnimal.scientific_name));
```

Comment:
you need to leave a space in between your quotes, else your output will be squashed against each other.

zooAnimal.animal_name + **'put a space inside these quotes'** +zooAnimal.scientific_name